src/content/docs/zh-cn/recipes/i18n.mdx

```diff
@@ -0,0 +1,332 @@
+---
+title: 添加 i18n 功能
+description: 通过动态路由和文档集来让你的 Astro 站点支持国际化。
+type: recipe
+i18nReady: true
+---
+import FileTree from '~/components/FileTree.astro'
+import StaticSsrTabs from '~/components/tabs/StaticSsrTabs.astro'
+
+尽管 Astro 没有内置的国际化功能，但你也可以构建你自己的国际化解决方案。
+
+本节示例中，你将看到如何使用文档集和动态路由来支持不同语言。
+
+假设示例拥有不同语言的子路径，例如面向英语的 `example.com/en/blog` 和面向法语的 `example.com/fr/blog`。
+
+## 如何集成？
+
+
+### 为页面设置多语言
+
+1. 创建一个你想支持的语言的目录。例如，如果你打算支持英语和法语，那么是创建 `en/` 和 `fr/` 目录：
+    <FileTree>
+    - src/
+      - pages/
+        - **en/**
+          - about.astro
+          - index.astro
+        - **fr/**
+          - about.astro
+          - index.astro
+        - index.astro
+    </FileTree>
+    
+2. 设置 `src/pages/index.astro` 重定向到你的默认语言。
+
+    <StaticSsrTabs>
+      <Fragment slot="static">
+        ```astro
+        ---
+        // src/pages/index.astro
+        ---
+        <meta http-equiv="refresh" content="0;url=/en/" />
+        ```
+
+        这种方式通过 [meta refresh](https://en.wikipedia.org/wiki/Meta_refresh) 来实现且不会受限于你的部署方式；不过有些静态服务器也允许你通过自定义配置文件来让你的服务器重定向，这可能需要你进一步参考你所使用的部署平台文档。
+      </Fragment>
+      
+      <Fragment slot="ssr">
+        如果你正好使用 SSR 适配器，那么你也可以使用 [`Astro.redirect`](/zh-cn/core-concepts/routing/#dynamic-redirects) 来重定向到默认语言。
```

Comment:
There will be a broken link.

Reply:
@liruifengv Should I update this broken link content in this PR or use a new PR to update that?